src/boot/efi/util.h

```diff
@@ -116,14 +116,23 @@ static inline void unload_imagep(EFI_HANDLE *image) {
                 (void) BS->UnloadImage(*image);
 }
 
+/* Creates a EFI_GUID pointer suitable for EFI APIs. Use of const allows the compiler to merge multiple
+ * uses (although, currently compilers do that regardless). Most EFI APIs declare their EFI_GUID input
+ * as non-const, but almost all of them are in fact const. */
+#define MAKE_GUID_PTR(name) ((EFI_GUID *) &(const EFI_GUID) name##_GUID)
```

Comment:
I'd shorten this to `GUID_PTR()`

Reply:
I certainly wouldn't mind too much, but do we do similar stuff in userspace that produces compound literals? The "make" conveys that something is created/"allocated" while `GUID_PTR` sounds like a cast to me…

More opinions on this would be nice.